client/components/jetpack/daily-backup-status/status-card/backup-successful.jsx

```diff
@@ -101,20 +100,18 @@ const BackupSuccessful = ( {
 			<div className="status-card__meta">{ meta }</div>
 			{ isMultiSite && (
 				<div className="status-card__multisite-warning">
-					<div className="status-card__multisite-warning-title">
-						{ preventWidows( translate( 'This site is a WordPress Multisite installation.' ) ) }
-					</div>
+					<h2 className="status-card__multisite-warning-title">
+						{ translate( 'This site is a WordPress Multisite installation.' ) }
+					</h2>
 					<p className="status-card__multisite-warning-info">
-						{ preventWidows(
-							translate(
-								'Jetpack VaultPress Backup for Multisite installations provides downloadable backups, no one-click restores. ' +
-									'For more information {{ExternalLink}}visit our documentation page{{/ExternalLink}}.',
-								{
-									components: {
-										ExternalLink: <ExternalLink href={ multiSiteInfoLink } target="_blank" icon />,
-									},
-								}
-							)
```

Comment:
I had to fudge some conditions to make this one display outside of the multisite context:

<img width="317" alt="Screenshot 2025-05-08 at 3 31 03 pm" src="https://github.com/user-attachments/assets/cfd942bd-19fc-4141-bed6-307949423813" />

It's not clear that the title actually looks better with pretty wrapping though 😅